### ISSUE TITLE
Any new development?

### ISSUE THREAD

**Issue by minerkapwinnnng** [fabulouspanda/MacMiner#20]
Good afternoon.  Given the current state of crypto, have you considered doing any updates to your software?  I've got an older Mac with a newer, pretty stout video card, and it would be awesome to be able to take advantage of the increased processing power.  

Also, aside from the main pages on your site and forum, all of the discussion pages are no longer found.  

Thanks.

**Comment by fabulouspanda** [comment]
Hey,

What currencies are you interested in mining? I'm not following alt coins as much as I was so not totally sure there's much suitable for Mac, but I stopped actively developing when the only coins you could productively mine on a Mac were harsh enough on the hardware that I figured it wasn't really for people who weren't comfy at the command line. Also the graphics drivers for Macs are so bad you'd be way better off booting a different OS if you're doing it seriously. Also, Apple are starting to use their own CPU/GPU so there's unlikely to be a future in it unless there are some currencies using algorithms that would benefit from Apple's own hardware.

Where are the links to discussion pages from that are broken? The forum moved a long time ago so it may be that, I'll fix the links if it's online but if it's in the app I won't bother updating it just for the documentation I'm afraid.

edit: this is the current forum url https://fabulouspanda.com/forum/ , please let me know if you're using an older one like the .co.uk or something and I'll start redirecting it

**Comment by minerkapwinnnng** [comment]
Looks like the discussion pages are working now.

I've just been running scrypt for the last few months, before it really took off with Doge...

The GPU Miner seems to pick up the GPU okay.  Its actually a Radeon RX 580.  Mainly got it for the HDMI outputs so I could have an HDMI extender with USB and monitor in my workshop without keeping the actual MacPro in there...  (I've got one of the older mid 2012 models, but it's got 12 cpu cores, 32GB RAM and now this upgraded video card.)  Gets kinda warm and dirty in the shop working on a project...  Then, I found my kids and I can run Velicidrone (drone racing simulator) on it so we can practice without crashing our actual drones...  Since it's left on 24x7, figured I'd have it do something useful, even if it isn't super profitable.  

If you aren't up to doing any updates, I understand that the user base isn't exactly huge.  I certainly appreciate you getting back to me this morning either way!  

-Ryan

**Comment by VicPT** [comment]
I'm very interested as well. Using a Mac Pro 5.1 with 2x RX 580 8gb.

I managed to get Mac Miner to work with the decred algorithm using a Nicehash pool. But it seems that I can't get much more to work on it as it has outdated algo's for mining. 

Something like DaggerHashimoto aka ETHASH would be nice and support for Binance pool for example.

My Mac is on 24/7 and most of the time my 2nd GPU is idle.

**Comment by timgjr** [comment]
> Hey,
> 
> What currencies are you interested in mining? I'm not following alt coins as much as I was so not totally sure there's much suitable for Mac, but I stopped actively developing when the only coins you could productively mine on a Mac were harsh enough on the hardware that I figured it wasn't really for people who weren't comfy at the command line. Also the graphics drivers for Macs are so bad you'd be way better off booting a different OS if you're doing it seriously. Also, Apple are starting to use their own CPU/GPU so there's unlikely to be a future in it unless there are some currencies using algorithms that would benefit from Apple's own hardware.
> 
> Where are the links to discussion pages from that are broken? The forum moved a long time ago so it may be that, I'll fix the links if it's online but if it's in the app I won't bother updating it just for the documentation I'm afraid.
> 
> edit: this is the current forum url https://fabulouspanda.com/forum/ , please let me know if you're using an older one like the .co.uk or something and I'll start redirecting it

True, but now with external GPU's you can blow out the eGPU and not the Mac. :) I have an eGPU was hoping to use this to farm a little at night when not using my Mac..
